templates/taxi/index.html

```diff
@@ -10,4 +10,5 @@ <h2>Dynamic content</h2>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
   </ul>
+    <p>You have visited this page {{ num_visits }} time{{ num_visits|pluralize }}</p>
 {% endblock %}
```

Comment:
your project doesn't meet these two conditions:
1. If the driver clicks on username - the corresponding Driver detail page must open.
2. In Driver's list view add (Me), if this is a current user: